package.json

```diff
@@ -20,11 +20,13 @@
     "gulp-less": "^3.0.3",
     "gulp-load-plugins": "^1.1.0",
     "gulp-imagemin": "^2.3.0",
+    "gulp-modernizr": "^1.0.0-alpha",
     "gulp-mq-remove": "0.0.2",
     "gulp-notify": "^2.2.0",
     "gulp-sitespeedio": "^0.0.8",
     "gulp-sourcemaps": "^1.5.2",
     "gulp-replace": "^0.5.3",
+    "gulp-uglify": "^1.5.1",
```

Comment:
Why is `gulp-uglify` a dependency when `gulp-rename` is a dev-dependency? Are we all the modules in gulp build moving to dependencies since they're required to build the site?